docs/reference/announcements.md

```diff
@@ -19,6 +19,10 @@ were deprecated in `8.4`. Please use the dedicated Camunda Identity properties o
 
 As of the 8.4 release, Camunda is compatible with Elasticsearch 8.9+ and no longer supports older Elasticsearch versions. See [supported environments](/docs/reference/supported-environments.md).
 
+### Support for Amazon OpenSearch
+
+As of the 8.4 release, Zeebe, Operate, and Tasklist are now compatible with [Amazon OpenSearch](https://aws.amazon.com/de/opensearch-service/) 2.5.x. Note that using Amazon OpenSearch requires setting up a new Camunda installation. A migration from previous versions or Elasticsearch environments is currently not supported.
```

Comment:
"requires setting up a new Camunda installation" -- can you explain this? Do you mean across components, SaaS, SM? This will help me link to the appropriate documentation to execute the installation.

Reply:
@christinaausley I think pointing to https://docs.camunda.io/docs/self-managed/platform-deployment/overview/ should be sufficient since they could following links further from there. This is particularly relevant for SM. Does this help?